app/src/navigators/RootStack.tsx

```diff
@@ -76,189 +62,17 @@ const RootStack: React.FC = () => {
     })
   }, [declinedProofs, agent, store.preferences.useDataRetention])
 
-  const lockoutUser = useCallback(async () => {
-    if (agent && store.authentication.didAuthenticate) {
-      // make sure agent is shutdown so wallet isn't still open
-      removeSavedWalletSecret()
-
-      try {
-        await agent.wallet.close()
-
-        logger.info('Closed agent wallet')
-      } catch (error) {
-        logger.error(`Error closing agent wallet, ${error}`)
-      }
-
-      dispatch({
-        type: DispatchAction.DID_AUTHENTICATE,
-        payload: [{ didAuthenticate: false }],
-      })
-
-      dispatch({
-        type: DispatchAction.LOCKOUT_UPDATED,
-        payload: [{ displayNotification: true }],
-      })
-    }
-  }, [agent, dispatch, logger, removeSavedWalletSecret, store.authentication.didAuthenticate])
-
   useEffect(() => {
     loadState(dispatch)
       .then(() => {
         dispatch({ type: DispatchAction.STATE_LOADED })
       })
-      .catch((err) => {
-        const error = new BifoldError(t('Error.Title1044'), t('Error.Message1044'), err.message, 1001)
+      .catch((err: unknown) => {
```

Comment:
Pourquoi on ne type pas err: Error directement ici ?

Reply:
Voir la raison ici: https://www.typescriptlang.org/play/?ts=5.7.2#example/unknown-in-catch